### PR TITLE
Update Project Structure Description & Fix Code Style

### LINKED ISSUE #1599: Documentation bug: outdated directories
I believe that some of the the directories specified in the [Project Structure](https://markbind.org/devdocs/devGuide/design/projectStructure.html) section of the developer's guide might be outdated: 
<br>
Screenshot 2021-06-27 at 12 22 32 AM 
The CLI application should reside in the `packages/cli/` directory.

<br>
Screenshot 2021-06-27 at 12 23 54 AM
There is no src/Parser directory in the core library.

### PR DESCRIPTION
**Pull request by kaixin-hc**
**What is the purpose of this pull request?**

- [X] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] Others, please explain:

Fixes #1599 .

**Overview of changes:**
* Fix markdown to match code style ("Use generic numbering for ordered lists." + new line after header
* Update section under "The core library mainly houses:"
* Update the location of the CLI interface application.

**Anything you'd like to highlight / discuss:**
I'm new to the project, I hope I described the project structure accurately! I referred heavily to the architecture page in the developer guide and thought about what might have helped me when I started examining the project.

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**
Update projectStructure.md

---

**Checklist:** :ballot_box_with_check:

- [X] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [ ] Linked all related issues
- [ ] No unrelated changes

### CODE REVIEW

File: docs/devGuide/design/projectStructure.md [hunk 1]
```diff
@@ -31,21 +31,25 @@ The MarkBind project is developed in a monorepo ([MarkBind/markbind](https://git
 ### MarkBind core library
 
 **The core library mainly houses:**
-1. Various key functionalities in processing a _singular_ page inside `src/Parser` that transforms MarkBind syntax into valid html output.
 
-2. The `src/Page` model, which binds the functionalities exposed by `Parser` to generate a _single_ page.
+1. Functions and libraries used to parse and process MarkBind into usable output are stored in `src` The architecture described in [Architecture](../design/architecture).html) is contained here. A brief rundown of what it includes:
```
Comment (jonahtanjz) [inline docs/devGuide/design/projectStructure.md:35]:
```suggestion
1. Functions and libraries used to parse and process MarkBind into usable output are stored in `src` The architecture described in [Architecture](../design/architecture) is contained here. A brief rundown of what it includes:
```

**Review by jonahtanjz — CHANGES_REQUESTED** [review]
Thanks for updating the docs :) Looks quite detailed and accurate to me 👍

**Comment by kaixin-hc** [comment]
Thanks Jonah! I've made the change you suggested. The commit after that is to make sure the lists obey grammar conventions + rearranging the first list so that it has the same order as the section below for clarity.

**Review by jonahtanjz — APPROVED** [review]
Thanks for making the changes :) 

LGTM 👍

**Pull request merged**